.gitignore

```diff
@@ -82,6 +82,7 @@ celerybeat-schedule
 # virtualenv
 venv/
 ENV/
+environment
```

Comment:
I don't think we need this change here

Reply:
Thanks for the review let me make the change then  update you

Reply:
Already fixed the issue